abap-code-review/README.md

```diff
@@ -0,0 +1,25 @@
+# ABAP Code Review Guide
+ABAP Code Review Guide is a collection of ideas, tools, and approaches for implementing ABAP code reviews in various landscapes.
+
+The document is provided as open source under the [CC license](../LICENSE), suggestions and bugfixes are welcome.
```

Comment:
Why is an additional license needed?

Reply:
the link is to the existing license, https://github.com/xtough/styleguides/blob/main/abap-code-review/README.md

Reply:
...in fact to https://github.com/xtough/styleguides/blob/main/LICENSE, which would become https://github.com/SAP/styleguides/blob/main/LICENSE after merging the PR.